tools/topology/topology1/sof-tgl-max98357a-rt5682.m4

```diff
@@ -146,7 +146,11 @@ define(DMIC_PIPELINE_KWD_ID, `12')
 define(DMIC_DAI_LINK_16k_ID, eval(BOARD_DMIC_BE_ID_BASE + 1))
 define(DMIC_PIPELINE_48k_CORE_ID, `1')
 
-ifdef(`RTNR', `define(`DMICPROC', rtnr)', `')
+ifdef(`RTNR',
```

Comment:
Its probably best to have a comment here stating that this is for RTNR version xyz (and wont work with older versions etc).

Reply:
Updated. Thanks!